### PR TITLE
EverHot

### PR DESCRIPTION
**Pull request by BL0bb3r44**
# Your checklist for this pull request

## About your game

What is your game about? 
My game is a combination of the rhythm indie game everhood and superhot goal of the game is to survive for as long as possible while dodging attacks that you create by moving if you move left there will be a attack from the left and same for right and up and down, so if you move more you can make it harder and less you can make it easier. If you get hit you are dead.

How to play your game?
> Example  : Press WASD to move J to restart, survive.

## Code
Check off the items that are true.
- [ check] The game was made using the [Sprig editor](https://editor.sprig.hackclub.com/).
- [ check] The game is placed in the in the [`/games` directory](https://github.com/hackclub/sprig/tree/main/games).
- [ check] The code is significantly different from all other games in the [Sprig gallery](https://sprig.hackclub.com/gallery) (except for games labeled "demo").
- [ check] The game runs without errors. 
- [check ] The name of the file/game contains only alphanumeric characters, `-`s, or `_`s.
- [ check] The game name is not the same as the others from [gallery](https://sprig.hackclub.com/gallery)

## Image (If an image is used)

- [ check] The image is in the [`/games/img` directory](https://github.com/hackclub/sprig/tree/main/games/img).
- [ check] The name of the image matches the name of your file. Example: `sokoban_plus.js` -> `sokoban_plus.png`.
what just so you know in the image thumbnail i did not at. the _ in between it is 1 word. Also great thing you guys are doing cant wait to get my hands on a sprig.
> Thanks for PR!

### CODE REVIEW

**Comment by LucasHT22** [comment]
just one thing:
is it possible to continue playing after losing

**Comment by BL0bb3r44** [comment]
@LucasHT22 yes sorry I forgot to add that it should be j how can I update the file

**Comment by BL0bb3r44** [comment]
I have the new file but I can't put in the comments because it is a js file

**Comment by LucasHT22** [comment]
> I have the new file but I can't put in the comments because it is a js file

make changes in: `Files changed` > `...` > `edit`

**Comment by BL0bb3r44** [comment]
Ok Thanks I added the code to restart

**Comment by BL0bb3r44** [comment]
@LucasHT22 how will I know if I get a sprig console? Just so I do not miss an email or something?

**Comment by LucasHT22** [comment]
> @LucasHT22 how will I know if I get a sprig console? Just so I do not miss an email or something?

when I approve, I'll send you a forms here

**Comment by LucasHT22** [comment]
I found a bug:

https://user-images.githubusercontent.com/88567622/201415276-544ece88-2993-4887-91bf-3705907bd56a.mp4

**Comment by BL0bb3r44** [comment]
oh whoops thats dumb I will fix that sorry give me one second

**Comment by BL0bb3r44** [comment]
@LucasHT22  I Fixed it I added an isAlive variable and set it to false when you die, and put the movement in a if isAlive = true.

**Comment by LucasHT22** [comment]
> @LucasHT22 I Fixed it I added an isAlive variable and set it to false when you die, and put the movement in a if isAlive = true.

you don't need to say sorry! mistakes are normal:)

File: games/ever_hot.js [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    getFirst(player).x += 1
+    moveAll();
+    addSprite(2, 1, baddude)
+    points += 1
+  }
+});
+
+afterInput(() => {
+  if (tilesWith(player, baddude).length > 0){
+    addText("YouLose", {
```
Comment (LucasHT22) [inline games/ever_hot.js:178]: just one thing, add a space here in `YouLose`

Comment (BL0bb3r44) [reply games/ever_hot.js]: done

**Review by LucasHT22 — APPROVED** [review]
Looks good to me! Congrats:)
If you're a teen, you can get your sprig here! https://sprig-order.hackclub.dev/

out of curiosity, how did you meet sprig?

**Comment by BL0bb3r44** [comment]
@LucasHT22 my dad told me about it and i enjoy coding, games, and free stuff so I learned java script and made a game.

**Comment by LucasHT22** [comment]
nice!

**Pull request merged**